wifimgr.py

```diff
@@ -202,11 +206,19 @@ def handle_configure(client, request):
                         <span style="color: #ff0000;">
                             ESP successfully connected to WiFi network %(ssid)s.
                         </span>
-                    </h1>
+                    </h1>""" % dict(ssid=ssid)
+        if link_to_next_webui:
+            response += """\
+                    <p style="text-align: center;">
+                        <a href="http://%(ip)s/">To new Interface [%(ip)s]</a><br>
```

Comment:
Thanks for updating this. But there is a slight problem now: the new IP is only shown IF link_to_next_webui is True (which only makes sense if you have a http web interface running on the new IP).

If you don't have a http web interface running there, you want to set this to False, but then it also won't display the IP address in plain text - but you might need it, if you run something else on that IP.

Reply:
My idea: Add an extra variable that if `True` shows the IP on the success page and leave `link_to_next_webui` as is. Do you agree ?

Reply:
Or just always show it, because "why not"?